acquia_lift.admin.wizard.inc

```diff
@@ -386,15 +414,34 @@ function acquia_lift_personalize_campaign_wizard_targeting_alter(&$form, &$form_
     );
   }
 
-  // Add a new audience.
-  $form['audiences']['add'] = array(
-    '#type' => 'container',
-    '#theme' => 'acquia_lift_card',
-    '#collapsible' => TRUE,
-    '#collapsed' => !empty($option_set->targeting),
-    '#title' => t('Add target audience'),
-  );
-  $form['audiences']['add']['details'] = _acquia_lift_personalize_campaign_wizard_targeting_audience($form, $form_state, $agent_data, 'add');
+  // If the agent has been started, then there is the possibility of unassigned
+  // variation options that need to be assigned to a region in order to be
+  // available for assignment via drag and drop.
+  if ($status !== PERSONALIZE_STATUS_NOT_STARTED) {
+    $assigned_options = array();
+    foreach ($saved_targeting as $audience_id => $options) {
+      $assigned_options = array_merge($assigned_options, $options);
+    }
+    $unassigned_options = array_diff_key($variation_options, array_flip($assigned_options));
```

Comment:
This is the display of unassigned options that get shown when variations are added to an option set after the agent is started.  Since they are no longer automatically assigned to everyone else - then they need to appear in their own special "unassigned" area so that they are still in the UI to drag to an audience.